src/herder/TxSetFrame.cpp

```diff
@@ -274,22 +297,22 @@ TxSetFrame::surgePricingFilter(Application& app)
                 cur->clear();
             }
         }
-        mTransactions = std::move(updatedSet);
-        sortForHash();
+
+        transactions = std::move(updatedSet);
+        // sortForHash();
     }
 }
 
 bool
-TxSetFrame::checkOrTrim(Application& app,
-                        std::vector<TransactionFrameBasePtr>& trimmed,
+TxSetFrame::checkOrTrim(Application& app, TxSetFrame::Transactions& trimmed,
```

Comment:
Seems like trimming logic is gone now, isn't it? Previously this mutated the tx set when `justCheck` is `false`.
I think logically there should be a bool method for check and a method that returns a `TxSetFrame` for trim (maybe with partially shared implementation).

Reply:
The `checkOfTrim` method has been moved and renamed to `getInvalidTxList` in `TxSetUtils` which is a lot clearer on what it supposed to do. The old "trimInvalid" function (which is only used 1-2 places in the old production code), is now replace by a call to `getInvalidTxList` followed by `removeTxs` passing in the invalid set. In the `HerderTest.cpp` there is now a free function `trimInvalid` that imitates the old function and only used in tests.